src/app/visual/lookup/lookup-visual.component.html

```diff
@@ -6,71 +6,143 @@
   class="app-screenshot"
   id="lookup-visual"
 >
-  <form [formGroup]="form" novalidate>
+  <form
+    novalidate
+    [formGroup]="friendsForm"
+  >
+      <div class="sky-form-group">
+        <label
+          class="sky-control-label"
+          id="my-friends-label"
+        >
+          Who are your friends?
+        </label>
+        <sky-lookup
+          ariaLabelledBy="my-friends-label"
+          formControlName="friends"
+          idProperty="id"
+          placeholderText="Type a person's name..."
+          [data]="people"
+          [debounceTime]="1000"
+        >
+        </sky-lookup>
+      </div>
 
-    <div class="sky-form-group">
-      <label
-        id="my-friends-label"
-        class="sky-control-label">
-        Who are your friends?
-      </label>
-      <sky-lookup
-        ariaLabelledBy="my-friends-label"
-        formControlName="friends"
-        idProperty="id"
-        placeholderText="Type a person's name..."
-        [data]="people"
-        [debounceTime]="1000"
+      <p>
+        <button
+          class="sky-btn sky-btn-default"
+          id="btn-disable-lookup"
+          type="button"
+          (click)="disableLookup()">
```

Comment:
Closing caret needs to be on a new line.

Reply:
Fixed